plugins/design.cpp

```diff
@@ -0,0 +1,271 @@
+#include <bitset>
+#include <cstdint>
+#include <iostream>
+#include <map>
+#include <string>
+#include <vector>
+
+#include "ColorText.h"
+#include "Debug.h"
+#include "LuaTools.h"
+#include "PluginManager.h"
+#include "df/graphic_viewportst.h"
+#include "df/world.h"
+#include "modules/Gui.h"
+#include "modules/Persistence.h"
+#include "modules/Screen.h"
+#include "modules/World.h"
+
+DFHACK_PLUGIN("design");
+DFHACK_PLUGIN_IS_ENABLED(is_enabled);
```

Comment:
since the plugin isn't enableable, you don't need this line. The services offered by this plugin will be accessible as long as it is loaded. it doesn't need an explicit call to `enable` to function.